src/Renderer.php

```diff
@@ -217,14 +217,15 @@ public function inlineJson($values)
      * @return array
      *
      */
-    private function getBreakStripeHeading($stripe, $index) {
+    private function getBreakStripeHeading($stripe) {
         if (empty($stripe['heading']['html'])) {
             return null;
         } else {
+            $stripeId = $stripe['id'] ?? 0;
             return [
                 'level' => 1,
                 'headingText' => strip_tags($stripe['heading']['html']),
-                'id' => 'vssl-stripe--break--heading-' . $index
+                'id' => "vssl-stripe--break-$stripeId--heading"
```

Comment:
Oh, okay, probably don't add an ID like this unless you have one coming out of the API? As it is now, if the API isn't updated (e.g. currently) a bunch of elements could have the same ID (`vssl-stripe--break-0--heading`) and they should be unique.

We can also (this is a note for myself) release this PR as a breaking change/major version, so that old sites don't get updated to it by accident. But still worth handling that case, I think.

Reply:
I updated to check for the 'heading_id' value that gets computed on the api side and only add that as the id entry here if it isn't empty 👌